src/satellite/config.ts

```diff
@@ -34,74 +34,88 @@ export const satelliteDefaults: SatelliteOpts = {
   minSnapshotWindow: 40
 }
 
-export interface SatelliteClientOpts {
-  app: string
-  env: string
-  token: string
-  port: number
-  address: string
-  timeout: number
-  pushPeriod: number
-}
-
 export const satelliteClientDefaults = {
+  env: "default",
   timeout: 3000,
-  pushPeriod: 500
+  pushPeriod: 500,
+  insecure: false
 }
 
-export interface SatelliteClientOverrides {
+const baseDomain = "electric-sql.com"
+
+export interface SatelliteClientOpts {
   app: string
-  env: string
+  env?: string
   token: string
+  host: string
   port: number
-  address: string
   timeout?: number
   pushPeriod?: number
+  insecure?: boolean
 }
 
+
 // Config spec
 export interface ElectricConfig {
   app: string
-  env: string
+  env?: string
   token: string
   migrations?: Migration[],
   replication?: {
-    address: string
+    host: string
```

Comment:
We need to make sure this change is reflected in simple client app, that we use in integration tests for electric

Reply:
This needs to be done for the release.

I am going to open a ticket for the release and point to this comment.

Reply:
Opened [VAX-465](https://linear.app/vaxine/issue/VAX-465/release-032)